jscomp/bsb/bsb_package_specs.ml

```diff
@@ -131,8 +131,7 @@ let package_flag ({format; in_source } : spec) dir =
     (Ext_string.concat3
        (string_of_format format)
        Ext_string.single_colon
-       (if in_source then dir else
-        prefix_of_format format // dir))
+         dir)
```

Comment:
I think you still want to support what these lines where doing, insource builds or building inside the prefix-ed sub directory (like es6, commonjs etc...).